Marlin/ultralcd_implementation_hitachi_HD44780.h

```diff
@@ -442,175 +442,173 @@ Possible status screens:
        |F100%  SD100% T--:--|
        |Status line.........|
 */
-static void lcd_implementation_status_screen()
-{
-    int tHotend=int(degHotend(0) + 0.5);
-    int tTarget=int(degTargetHotend(0) + 0.5);
+static void lcd_implementation_status_screen() {
+  int tHotend=int(degHotend(0) + 0.5);
+  int tTarget=int(degTargetHotend(0) + 0.5);
 
 #if LCD_WIDTH < 20
-    lcd.setCursor(0, 0);
-    lcd.print(itostr3(tHotend));
-    lcd.print('/');
-    lcd.print(itostr3left(tTarget));
-
-# if EXTRUDERS > 1 || TEMP_SENSOR_BED != 0
+  lcd.setCursor(0, 0);
+  lcd.print(itostr3(tHotend));
+  lcd.print('/');
+  lcd.print(itostr3left(tTarget));
+  
+  #if HOTENDS > 1 || TEMP_SENSOR_BED != 0
     //If we have an 2nd extruder or heated bed, show that in the top right corner
     lcd.setCursor(8, 0);
-#  if EXTRUDERS > 1
-    tHotend = int(degHotend(1) + 0.5);
-    tTarget = int(degTargetHotend(1) + 0.5);
-    lcd.print(LCD_STR_THERMOMETER[0]);
-#  else//Heated bed
-    tHotend=int(degBed() + 0.5);
-    tTarget=int(degTargetBed() + 0.5);
-    lcd.print(LCD_STR_BEDTEMP[0]);
-#  endif
-    lcd.print(itostr3(tHotend));
-    lcd.print('/');
-    lcd.print(itostr3left(tTarget));
-# endif//EXTRUDERS > 1 || TEMP_SENSOR_BED != 0
+    #if HOTENDS > 1
+      tHotend = int(degHotend(1) + 0.5);
+      tTarget = int(degTargetHotend(1) + 0.5);
+      lcd.print(LCD_STR_THERMOMETER[0]);
+    #else//Heated bed
+      tHotend=int(degBed() + 0.5);
+      tTarget=int(degTargetBed() + 0.5);
+      lcd.print(LCD_STR_BEDTEMP[0]);
+    #endif
+      lcd.print(itostr3(tHotend));
+      lcd.print('/');
+      lcd.print(itostr3left(tTarget));
+  #endif //HOTENDS > 1 || TEMP_SENSOR_BED != 0
 
 #else//LCD_WIDTH > 19
-    lcd.setCursor(0, 0);
-    lcd.print(LCD_STR_THERMOMETER[0]);
-    lcd.print(itostr3(tHotend));
-    lcd.print('/');
-    lcd.print(itostr3left(tTarget));
-    lcd_printPGM(PSTR(LCD_STR_DEGREE " "));
-    if (tTarget < 10)
-        lcd.print(' ');
-
-# if EXTRUDERS > 1 || TEMP_SENSOR_BED != 0
+  lcd.setCursor(0, 0);
+  lcd.print(LCD_STR_THERMOMETER[0]);
+  lcd.print(itostr3(tHotend));
+  lcd.print('/');
+  lcd.print(itostr3left(tTarget));
+  lcd_printPGM(PSTR(LCD_STR_DEGREE " "));
+  if (tTarget < 10)
+    lcd.print(' ');
+
+  #if HOTENDS > 1 || TEMP_SENSOR_BED != 0
     //If we have an 2nd extruder or heated bed, show that in the top right corner
     lcd.setCursor(10, 0);
-#  if EXTRUDERS > 1
-    tHotend = int(degHotend(1) + 0.5);
-    tTarget = int(degTargetHotend(1) + 0.5);
-    lcd.print(LCD_STR_THERMOMETER[0]);
-#  else//Heated bed
-    tHotend=int(degBed() + 0.5);
-    tTarget=int(degTargetBed() + 0.5);
-    lcd.print(LCD_STR_BEDTEMP[0]);
-#  endif
+    #if HOTENDS > 
```

Comment:
If `HOTENDS` is greater than... what? This needs a right operand.

    /ultralcd_implementation_hitachi_HD44780.h:485:19: error: operator '>' has no right operand